packages/data-point-codemods/transforms/__testfixtures__/hash-and-collection-execution-order.output.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable */
+/* eslint-disable prettier */
+
+const e1 = {}
+
+// prettier-ignore
+e1 = {
+  entities: {
+    'collection:nested': {
+      compose: [{
+        filter: '$a'
+      }, {
+        map: '$b'
+      }],
+
+      after: () => true
+    }
+  }
+}
+
+const e2 = {}
+
+// prettier-ignore
+e2['hash:example'] = {
+  'after': () => ({}),
+  'error': () => ({}),
+  'params': {},
+  'before': () => ({}),
+  value: {},
+  'inputType': 'object',
+
+  compose: [{
```

Comment:
jcodeshift adds the empty line before `compose`. not sure how to prevent that...

Reply:
That's interesting, and hard to ping down. This is the closest information I can find to it. https://github.com/benjamn/recast/issues/147

Is this a big deal to try to "fix" or are we OK with leaving it? I don't think it's worth the effort to solve this as any source files changed by the codemod might go through a linter/formatter by users anyway and leave/change these empty lines on their own.

Reply:
I don't think it's worth trying to fix

Reply:
OK, yeah, I agree.